modules/ROOT/examples/live-demos/url-conversion-relative-1/index.html

```diff
@@ -0,0 +1,4 @@
+<textarea id="url-conversion-relative-1">
+  <p>Some relative urls: <a href="../index.html">content</a>. <img src="{{logofordemos}}" alt="Tiny Logo" width="128" height="128" /></p>
+  <p>Some absolute urls: <a href="https://www.tiny.cloud/docs/demo/index.html">content</a>. <img src="{{logofordemos}}" alt="Tiny Logo" width="128" height="128" /></p>
```

Comment:
closest URL without an index page
https://www.tiny.cloud/tinymce/6/basic-example.html

Reply:
find and replace all